examples/user_guide/Plots_and_Renderers.ipynb

```diff
@@ -83,8 +83,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from holoviews.plotting.mpl import MPLRenderer\n",
-    "renderer = MPLRenderer.instance(dpi=120)"
+    "renderer = holoviews.plotting.mpl.MPLRenderer.instance(dpi=120)"
```

Comment:
An alternative would be to do `from holoviews.plotting import mpl as hpm` at the top, then `hpm.MPLRenderer.instance` here...